### PR TITLE
TA readme and changelog updates

### PR DESCRIPTION
**Pull request by maririos**
Part of https://github.com/Azure/azure-sdk-for-net/issues/11608

### CODE REVIEW

**Review by iscai-msft — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 ### Breaking changes
 - Removed `GraphemeLength` and `GraphemeOffset` from `CategorizedEntity`, `SentenceSentiment`, and `LinkedEntityMatch`.
 - `GraphemeCount` in `TextDocumentStatistics` has been renamed to `CharacterCount`.
+- In both `DetectedLanguage` property `SentimentScores` has been renamed to `ConfidenceScores`.
```
Comment (kinelski) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:7]: nit: In both `DetectedLanguage` **properties**...?

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
+
+### Added
+- Added `Text` property to `SentenceSentiment`.
+- `Warnings` property exist on each document-level response object returned from the endpoints. It is a list of `TextAnalyticsWarnings`.
```
Comment (kinelski) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:11]: nit: property exist => property exists

**Review by kinelski — APPROVED** [review]
LGTM.

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md (hunk 2 above)
Comment (kinelski) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:11]: Maybe rephrasing it to
`Warnings` properties added to each document-level response...

**Pull request merged**